src/Network/XMPP/XEP/Form.hs

```diff
@@ -2,6 +2,12 @@
 {-# LANGUAGE QuasiQuotes       #-}
 {-# LANGUAGE DataKinds         #-}
 
+-----------------------------------------------------------------------------
+-- |
+-- Copyright   :  (c) riskbook, 2020
+-- License     :  BSD-style (see the file libraries/base/LICENSE)
```

Comment:
Better to use `SPDX-License-Identifier:` field with the official name of the license.

Reply:
You're the second person to tell me [that](https://github.com/melpa/melpa/pull/7080#issuecomment-678807489), what does it mean?!

Reply:
That is the standardised field that should have the regulated name of the license, so it is not a free-style as above, but the standardised versions of supported licenses 🙂

Reply:
Please check if it's how you want.